hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/performance.md

```diff
@@ -94,6 +94,85 @@ on the client requirements.
 </property>
 ```
 
+## <a name="bulkdelete"></a> Improving delete performance through bulkdelete API.
+
+For bulk delete API spec refer to [BulkDelete](bulkdelete.html).
```

Comment:
generate site and make sure things line up, or just say "filesystem specification"